docker-compose.yml

```diff
@@ -22,23 +22,24 @@ services:
   kafka:
     restart: always
     image: wurstmeister/kafka:latest
+    ports:
+      - "9092:9092"
     depends_on:
       - zookeeper
     environment:
       KAFKA_ADVERTISED_HOST_NAME: kafka
+      KAFKA_ADVERTISED_PORT: 9092
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-    logging:
-      driver: none
 
   postgres:
     restart: always
     build: external/postgres
     ports:
-    - 5432:5432
+    - "5432:5432"
```

Comment:
I think it's better to only expose the REST port from the api gateway in order to isolate as much as possible the app's environment. Maybe to expose them through the development `docker-compose` file would be better.